modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/tasks/WatchTask.java

```diff
@@ -290,9 +293,25 @@ private static long _getBundleId(
 		return differences;
 	}
 
-	private static String _getReferenceInstallURL(File file) {
+	private static String _getReferenceInstallURL(
```

Comment:
Can we remove the keywords `static` and `final` from the private methods?

Then we can avoid passing in the `bundleSymbolicName` as a method parameter and just call `getBundleSymbolicName()`